chromium_src/extensions/browser/sandboxed_unpacker_unittest.cc

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/values.h"
+#include "brave/common/extensions/extension_constants.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace brave {
+bool BraveRewriteManifest(const std::string& extension_id,
+    base::DictionaryValue* manifest);
+}
+
+TEST(BraveSandboxedUnpacker, DoesNotRewriteForDifferentExtension) {
+  base::DictionaryValue manifest;
+  ASSERT_FALSE(
+      brave::BraveRewriteManifest("fffkpbalmllkdoenhmdmpbkajipdjfam",
+      &manifest));
+}
+
+TEST(BraveSandboxedUnpacker, RewriteForIPFSEDxtension) {
```

Comment:
Really minor, but is that a typo in the test name?

Reply:
Dxtension are decentralized extensions maybe? :P
Fixing.